src/cityreader/cityreader.py

```diff
@@ -1,6 +1,11 @@
+import csv
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
-
+class City:
+  def __init__(self, name, lat, lon):
+    self.name = name
+    self.lat = lat
+    self.lon = lon
```

Comment:
Really great job with this exercise :tada:

I think you will find it useful in the future to define `__repr__` and/or `__str__ `methods on your classes so you get clearer output for debugging. For example:

```
def __repr__(self):
    return f"<City name: {self.name}, lat: {self.lat}, lon: {self.lon}>"
```

`print(c)` will now give you `<City name: Seattle, lat: 47.6217, lon: -122.3238>`

That being said, you've demonstrated an ability to research a module you've never used before and have a perfect implementation, so again really nice job with this exercise 🏆